modules/contentbox/modules/contentbox-admin/views/authors/editor/password.cfm

```diff
@@ -1,55 +1,68 @@
 <cfoutput>
 <div class="tab-pane" id="change-password" style="min-height: 400px">
```

Comment:
I am not sure if this will be nice for the UI, since if you clicked on the `password` tab, now you will get nothing?